data/archiver/ingest_api.py

```diff
@@ -61,5 +67,42 @@ def get_staging_area(self):
             return self.submission['stagingDetails']['stagingAreaLocation']['value']
         return None
 
+    def get_all(self, url, entity_type, entities=[]):
```

Comment:
You could replace this file with ingest-client's ingest_api  module. You may reuse methods from there.

Reply:
the idea was to have this class extend the ingest api from ingest-client as i was anticipating some api calls that was not in ingest-client api, at least not when i checked, but these calls were later taken out. you are right, some methods could be reused from there and i will review this class.

Reply:
Keeping the custom ingest_api and not adding new ingest-client dependency for now, as new calls and response processing logics are being captured in here.